mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,17 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.deleteOrderById = (req, res) => {
+  Order.deleteOne({ _id: req.params.orderId })
+    .exec()
+    .then(result => {
+      if (result.deletedCount === 0) {
+        return res.status(404).send({ error: "No order found with that ID" });
```

Comment:
The error message should be consistent throughout the codebase. In other parts of the code, error messages are wrapped in `errorHandler(err)`. This should be consistent here as well. 

Change to: `return res.status(404).json({ error: errorHandler("No order found with that ID") });`